Support/Testing/CircleCI/Dockerfile

```diff
@@ -40,6 +40,35 @@ MAINTAINER Stephane Sezer <sas@fb.com>
 RUN apt-get update
 RUN apt-get install -y software-properties-common python-software-properties wget
 
+# Incase lldb needs built
+RUN apt-get install -y libz-dev
+RUN apt-get install -y swig
+RUN apt-get install -y ncurses-dev
+
+# Make the developer's life not suck
+RUN apt-get -y install sudo
```

Comment:
Same here, do the apt-add-repositry/apt-get update first, then do one big apt-get update for all the "utility" packages.

Reply:
Why do we need sudo? You should have root within the image itself, no?

Reply:
When I have to work on the container for prolonged periods of time I find it worth it to set up a user and import my `~/.dotfiles` settings. Things don't work properly without a non-root user. I'm tempted to add a `adduser facebook` for this purpose.

Reply:
For the time being, this isn't necessary a "testing Dockerfile" but rather a "testing and debugging Dockerfile". Maybe once we clear out the blacklists we can strip it towards being testing-only but for now minimizing this file serves literally 0 use or benefit at the cost of making debugging more cumbersome.